http/src/main/resources/reference.conf

```diff
@@ -66,7 +66,7 @@ gce {
   ]
   runtimeDefaults {
     machineType = "n1-standard-4"
-    diskSize = 50
+    diskSize = 20 # This is default size for just user disk
```

Comment:
default size for user disk when user is not opting to use persistent disk

Reply:
Is this for what we've been calling the 'boot disk'? I thought the max set of images be around ~30GB?

Reply:
no...boot disk is fixed to 50G because it needs to host the custom VM image we built. This disk is solely for storing user specific data....mounted in `/work`

Reply:
can we name it `bootDiskSize` for clarity?

Reply:
this is NOT boot disk......there's a separate entry for `bootDiskSize`...

Reply:
My bad. How about `persistentDiskSize` or `userDiskSize`? It's so easy to falsely assume.

Reply:
I think that can be confusing because `diskSize` is what we call it in request payload, and this config value is the default value for that field if the field is not specified

Reply:
Request is user-facing where we expose only the user disk (so no confusion there) whereas this config is internal where we have two types of disks so not sure if there should be an expectation of a one-to-one match between the config and the request parameter. Take it or leave it 😊